src/module/cp/peeper.py

```
@@ -73,10 +75,22 @@ def _validate(i: int, check_conf: dict):
+                if not isinstance(_uuid, str) or not _uuid:
+                    raise RuntimeError(
+                        f"`{field}[{j}]` must be a non-empty string "
+                        f"in peeper.configs[{i}]"
+                    )
+                try:
+                    parse_uuid(_uuid)
+                except ValueError as e:
+                    raise RuntimeError(
+                        f"Invalid uuid in `{field}[{j}]` of "
+                        f"peeper.configs[{i}]: {e}"
+                    )
```

Ruff (0.15.18)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)